packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -40,6 +40,7 @@ import {
 	NodeKind,
```

Comment:
Note for reviewers: the majority of the changes in this module are centered around the mapping between `view keys` and `stored keys`. Since the Flex domain now operates _strictly_ in terms of `stored keys`, we need to perform the appropriate mappings from view keys when accessing/editing data.